apm-sniffer/apm-sdk-plugin/mongodb-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/mongodb/v4/support/MongoInfoHelper.java

```diff
@@ -40,21 +41,26 @@
 import com.mongodb.internal.operation.UpdateOperation;
 import org.bson.BsonDocument;
 
+import java.lang.reflect.Field;
 import java.util.List;
 
 @SuppressWarnings({
-    "deprecation",
-    "Duplicates"
+        "deprecation",
+        "Duplicates"
```

Comment:
Have you done a local style check? You are changing the original source code format.
Wrong code styles would cause all CIs failing.

Reply:
This will slow down the extra time of tracing from 10 nanoseconds to 100+ ns or more, determined by how many `instanceof`s you have to run before the reflection.